problems/4.arrays.js

```diff
@@ -12,16 +18,21 @@ function firstAndLastAverage() {
 * @param {number[]} arr - The input array
 * @returns {number[]} - The duplicated array
 */
-function duplicateArray() {
-
-}
+function duplicateArray(arr=[]) {
+  let duplicate = arr.length 
+  duplicate.push()
+  }
+  
 
 /** Removes the first 3 elements from an array. If the array has fewer than 3 elements, return an empty array.
 * @param {any[]} arr - The input array
 * @returns {any[]} - The array with the first 3 elements removed
 */
-function removeFirstThreeElements() {
-
+function removeFirstThreeElements(arr) {
+if (arr.length >= 3){
+  arr.splice(0, 1, 2)}
```

Comment:
Splice returns the items that were removed, you want to return the arr after the items have been removed.
The way splice is written here, would replace the first element with the number 2. Please read how splice works on MDN.